mm2src/coins/lp_coins.rs

```diff
@@ -2741,6 +2750,7 @@ impl HttpStatusCode for WithdrawError {
             | WithdrawError::ContractTypeDoesntSupportNftWithdrawing(_)
             | WithdrawError::CoinDoesntSupportNftWithdraw { .. }
             | WithdrawError::NotEnoughNftsAmount { .. }
+            | WithdrawError::ChainIdRequired(_)
```

Comment:
I created this error but now I see it lacks a coin field instead of String field
Also possibly good to add a check at coin activation that if trezor is enabled then chain_id must be set in coins.
And this err apparently is not BAD_REQUEST but rather INTERNAL_SERVER_ERROR
(I can do this fix myself)

Reply:
I thought it's better to make the `chain_id` non optional instead but found that there are 2 coins in coin config [BIDR-BEP20](https://github.com/KomodoPlatform/coins/blob/2e6e232994dfc11880cfe4de094144d09e709fec/coins#L1666) and [VRA-ERC20](https://github.com/KomodoPlatform/coins/blob/2e6e232994dfc11880cfe4de094144d09e709fec/coins#L12220) with no chain id, I think these 2 coins configs should be fixed
c.c. @smk762 @cipig

Reply:
I will proceed with making `chain_id` non optional @dimxy

Reply:
Fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/1979/commits/def7e33cc9bc56866452c7772b16601afb608c1c